README.md

```diff
@@ -148,9 +148,10 @@ the build. If the CMake variables below are not specified, "main" branch
 of those repositories will be used. \<GIT\_BRANCH\_NAME\> should be the same
 as the Python backend repository branch that you are trying to compile.
 
-* triton-inference-server/backend: -DTRITON\_BACKEND\_REPO\_TAG=\<GIT\_BRANCH\_NAME\>
-* triton-inference-server/common: -DTRITON\_COMMON\_REPO\_TAG=\<GIT\_BRANCH\_NAME\>
-* triton-inference-server/common: -DTRITON\_COMMON\_REPO\_TAG=\<GIT\_BRANCH\_NAME\>
+* triton-inference-server/backend: -DTRITON_BACKEND_REPO_TAG=\<GIT\_BRANCH\_NAME\>
```

Comment:
`-DTRITON_BACKEND_REPO_TAG=<GIT_BRANCH_NAME>`  for better readability?

Reply:
I don't understand the difference.

Reply:
I should have been clear. Can you wrap it in "`"?

Reply:
Fixed.